dipy/io/utils.py

```diff
@@ -1,6 +1,8 @@
 """ Utility functions for file formats """
 import logging
+import os
 
+import dipy
```

Comment:
Can you replace this by `from dipy.io.stateful_tractogram import StatefulTractogram`